### DISCUSSION TITLE
About proof academic status

### DISCUSSION THREAD

**Discussion by jeongwoo903** [community/community#102523] [GitHub Education] [answered]
### Hi everyone,

Hi, I'm trying to create a student developer pack, so I've been trying to upload my school's ID card through various methods, but I keep getting rejected with the following message.

> Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.

> Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

I went into my GitHub billing information and set my information to be the same as my ID card, but due to the constant error, I would like to ask if it is possible to authenticate through other methods.

**Comment by faridnec** [comment, marked as answer]
Hey, this previously happened to me as well. I solved it by (before these steps make sure to fill in your billing info/address or any required info before proceeding to apply) restarting it and providing proof with the camera there not by uploading scanned documents. These steps solved my issue as they accepted it within 3 days. You might try this solution too!

**Reply by jeongwoo903** [reply]
나는 Billing information에 나의 이름을 ID 카드와 동일하게 한국어로 작성해 두었습니다. 하지만, 통과되지 않았습니다.

하지만 덕분에 Billing information에 영어로 이름을 작성하였고 이 문제를 해결할 수 있었습니다. 
감사합니다!

**Reply by faridnec** [reply]
You are welcome! 물론이죠 :)
